src/go/rpk/pkg/cli/cluster/partitions/move_status.go

```diff
@@ -0,0 +1,216 @@
+							filteredResponse = append(filteredResponse, r)
+						}
+					}
+				}
+			}
+			if len(filteredResponse) > 0 {
+				response = filteredResponse
+			}
+
+			sizeFn := func(size int) string {
+				if human {
+					return units.HumanSize(float64(size))
+				}
+				return strconv.Itoa(size)
+			}
+
+			f := func(rr *adminapi.ReconfigurationsResponse) interface{} {
+				var (
+					newReplica []int
+					oldReplica []int
+				)
+				nt := rr.Ns + "/" + rr.Topic
+				if rr.PartitionSize > 0 {
+					completion = rr.BytesMoved * 100 / rr.PartitionSize
+				}
+				for _, r := range rr.NewReplicas {
+					newReplica = append(newReplica, r.NodeID)
+				}
+				for _, r := range rr.PreviousReplicas {
+					oldReplica = append(oldReplica, r.NodeID)
+				}
+				return struct {
+					NT             string
+					PartitionID    int
+					MovingFrom     []int
+					MovingTo       []int
+					Completion     int
+					PartitionSize  string
+					BytesMoved     string
+					BytesRemaining string
+				}{
+					nt,
+					rr.PartitionID,
+					oldReplica,
+					newReplica,
+					completion,
+					sizeFn(rr.PartitionSize),
+					sizeFn(rr.BytesMoved),
+					sizeFn(rr.BytesLeft),
+				}
+			}
+
+			types.Sort(response)
+
+			const (
+				secMove      = "Partition movements"
+				secReconcile = "Reconciliation statuses"
+			)
+			sections := out.NewSections(
+				out.ConditionalSectionHeaders(map[string]bool{
+					secMove:      true, // we always print this section
+					secReconcile: all,  // we only print this section if -a is passed
+				})...,
+			)
+			sections.Add(secMove, func() {
+				headers := []string{"Namespace-Topic", "Partition", "Moving-from", "Moving-to", "Completion-%", "Partition-size", "Bytes-moved", "Bytes-remaining"}
+				tw := out.NewTable(headers...)
+				defer tw.Flush()
+				for _, tps := range response {
+					tw.PrintStructFields(f(&tps))
+				}
+			})
+
+			sections.Add(secReconcile, func() {
+				var j int
+				for _, p := range response {
+					fmt.Printf("%s\n", p.Ns+"/"+p.Topic+"/"+strconv.Itoa(p.PartitionID))
+					headers := []string{"Node-id", "Core", "Type", "Retry-number", "Revision", "Status"}
+					tw := out.NewTable(headers...)
+					for _, rs := range p.ReconciliationStatuses {
+						var row []interface{}
+						row = append(row, rs.NodeID)
+						for _, s := range rs.Operations {
+							row = append(row, s.Core, s.Type, s.RetryNumber, s.Revision, s.Status)
+						}
+						tw.Print(row...)
+					}
+					tw.Flush()
+					j++
+					if j < len(response) {
+						fmt.Println()
+					}
+				}
+			})
+		},
+	}
+
+	cmd.Flags().BoolVarP(&all, "print-all", "a", false, "Print internal states about movements for debugging")
+	cmd.Flags().BoolVarP(&human, "human-readable", "H", false, "Print the partition size in a human-readable form")
+	cmd.Flags().StringSliceVarP(&partitions, "partition", "p", nil, "Partitions to print the ongoing movements (repeatable)")
```

Comment:
Can this be clarified? What does "Partitions to print the ongoing movements (repeatable)" mean in this context?

Reply:
The command `move-status` prints `all` ongoing movements, the `partition` flag prints the ongoing movements for those partitions.

Reply:
Changed this to `Partitions to filter ongoing movements status (repeatable)` in 8c76b5f62b12ebc5a1affae0971a435c475d18ac